service/distro/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aqua-ipfs-distro"
-version = "0.5.26"
+version = "0.5.27"
```

Comment:
you don't need to increment that by hand, AFAIU

Reply:
I thought release-please would not catch this change, but I conducted the wrong experiment. Ok, I'll keep the old version